packages.dhall

```diff
@@ -113,14 +96,10 @@ let additions =
       , version =
           "v7.0.0"
       }
-  }
 -------------------------------
 -}
 let upstream =
-      https://github.com/purescript/package-sets/releases/download/psc-0.14.0-20210311/packages.dhall sha256:3da8be2b7b4a0e7de6186591167b363023695accffb98a8639e9e7d06e2070d6
-
-let overrides = {=}
-
-let additions = {=}
```

Comment:
It would be nice to drop these `spago` comments all together because they clutter the actual diffs :-P

Reply:
That's a fair point. I've stripped them out.